extensions/chromium/options/options.js

```diff
@@ -75,16 +75,14 @@ Promise.all([
       renderPreference = renderBooleanPref(prefSchema.title,
                                            prefSchema.description,
                                            prefName);
+    } else if (prefSchema.type === 'integer' && prefSchema.enum) {
+      // Most other prefs are integer-valued enumerations, render them in a
+      // generic way too.
```

Comment:
Unlike the boolean pref, each enum also requires an explicit HTML template in the document with ID `prefName + "-template"`. Can you document that too?

[ Previously, if a new enum is added without updating options.js/html, then the option would just not show up (and an error message be printed in the console, because of the `else`-block below). With this new change, an error will be thrown inside `getTemplate`, called via `renderEnumPref` (and the preference rendering will break). Proper documentation should prevent this from happening. ]